src/core/AutoRest/Plugins/AzureValidator.cs

```diff
@@ -80,11 +80,25 @@ private void LogValidationMessage(ValidationMessage validationMessage)
 
             var serviceDefinition = SwaggerParser.Load(file, fs);
             var validator = new RecursiveObjectValidator(PropertyNameResolver.JsonName);
-            
+            var docTypeInput = (await GetValue<string>("openapi-type"));
+            var docStateInput = (await GetValue<string>("merge-state"));
+
+            ServiceDefinitionDocumentType docType;
+            if (!Enum.TryParse<ServiceDefinitionDocumentType>(docTypeInput, true, out docType))
+            {
+                throw new Exception("Invalid Input for openapi-type: " + docTypeInput + ". Valid values are 'arm', 'data-plane' or 'default'.");
```

Comment:
do we need to include openapi-type in documentation somewhere, so folks know what values should be used and for what?